app/assets/javascripts/application.js.erb

```diff
@@ -185,7 +185,8 @@ class _LobstersFunction {
   }
 
   fetchURLTitle(button) {
-    const targetUrl = qS('#story_url').value;
+    const url_field = qS('#story_url');
```

Comment:
Btw fun fact, IDs that are syntactically valid JavaScript identifiers, automatically become variables pointing to their elements in the scope of the page. So we don't even need to use `qS('#story_url')`, we can access the element directly with `story_url`.

Reply:
I remember being burned by this Back In The Day when we used lots of globals in js, so I am leery of leaning on it now. :)